docs/book/src/user/quick-start.md

```diff
@@ -36,7 +36,7 @@ export KUBECONFIG=<...>
 
 [kind] is not designed for production use.
 
-**Minimum [kind] supported version**: v0.6.x
+**Minimum [kind] supported version**: v0.6.1
```

Comment:
I think we're now suggesting v0.7.+ ?

Reply:
Ok. I was going by the version used by CI scripts:
```
hack/ensure-kind.sh
24:MINIMUM_KIND_VERSION=v0.6.1
36:      curl -sLo "${GOPATH_BIN}/kind" https://github.com/kubernetes-sigs/kind/releases/download/${MINIMUM_KIND_VERSION}/kind-linux-amd64
46:  if ! [[ "${kind_version}" =~ ${MINIMUM_KIND_VERSION} ]]; then
49:Requires ${MINIMUM_KIND_VERSION} or greater.
50:Please install ${MINIMUM_KIND_VERSION} or later.

scripts/ci-integration.sh
26:KIND_VERSION="v0.6.1"
41:   wget "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-${GOOS}-${GOARCH}" \
```
Do these scripts need to be updated to 0.7.0? (I can open a separate PR)

Reply:
Looking at the code, right now we're using 0.7.1, maybe let's match that?

Reply:
(CAPD uses kind as a library I meant)

Reply:
Changed to 0.7.0.